experimental/iterators/tests/hash_join_operator_test.cc

```diff
@@ -0,0 +1,122 @@
+#include <gtest/gtest.h>
+
+#include <algorithm>
+#include <vector>
+
+#include "operators/column_scan.h"
+#include "operators/hash_join.h"
+
+TEST(HashJoinTest, SingleColumnKey) {
+  std::vector<int32_t> leftKeys = {1, 2, 1, 2, 5};
+  std::vector<int32_t> leftValues = {1, 1, 2, 2, 5};
+  std::vector<int32_t> rightKeys = {6, 1, 2, 1, 2};
+  std::vector<int32_t> rightValues = {6, 3, 3, 4, 4};
+
+  auto leftScan = makeColumnScanOperator(leftKeys, leftValues);
+  auto rightScan = makeColumnScanOperator(rightKeys, rightValues);
+  auto hashJoin = MakeHashJoinOperator<1>(&leftScan, &rightScan);
+
+  using ResultTuple = decltype(hashJoin)::OutputTuple;
+
+  // Consume result of hashJoin
+  hashJoin.open();
+  std::vector<ResultTuple> result;
+  while (const auto tuple = hashJoin.computeNext()) {
+    result.emplace_back(tuple.value());
+  }
+
+  // Compare with correct result
+  std::vector<ResultTuple> referenceResult = {
+      //
```

Comment:
spurious `//` here and below?

Reply:
This was actually intentional to make `clang-format` split it in two lines (which are similar to each other) rather than three. Turns out that just the second `//` is enough, which is what I do now. Is that frowned upon or something that should be commented?

Reply:
We generally just do:
```
// clang-format off
... code to format ourselves.
// clang-format on
```
on the bigger block we want to control.

This is fine too, no need to sweat it.